newIDE/app/src/MarketingPlans/MarketingPlanSingleDisplay.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+
+import * as React from 'react';
+import {
+  type MarketingPlan,
+  type GameFeaturing,
+  type Game,
+} from '../Utils/GDevelopServices/Game';
+import { Column } from '../UI/Grid';
+import Paper from '../UI/Paper';
+import Text from '../UI/Text';
+import GDevelopThemeContext from '../UI/Theme/GDevelopThemeContext';
+import MarketingPlanFeatures from './MarketingPlanFeatures';
+import usePurchaseMarketingPlan from './UsePurchaseMarketingPlan';
+import {
+  getRequirementsErrors,
+  isMarketingPlanActive,
+} from './MarketingPlanUtils';
+
+const styles = {
+  container: {
+    borderRadius: 8,
+    padding: 8,
+  },
+};
+
+type Props = {|
+  game: Game,
+  marketingPlan: MarketingPlan,
+  gameFeaturings: ?(GameFeaturing[]),
+  fetchGameFeaturings: () => Promise<void>,
+|};
+
+const MarketingPlanSingleDisplay = ({
+  game,
+  marketingPlan,
+  gameFeaturings,
+  fetchGameFeaturings,
+}: Props) => {
+  const gdevelopTheme = React.useContext(GDevelopThemeContext);
+
+  const activeGameFeaturings: ?(GameFeaturing[]) = React.useMemo(
+    () => {
+      if (!gameFeaturings) return null;
+
+      return gameFeaturings.filter(
+        gameFeaturing => gameFeaturing.expiresAt > Date.now() / 1000
+      );
+    },
+    [gameFeaturings]
+  );
+
+  const { onPurchaseMarketingPlan } = usePurchaseMarketingPlan({
+    game,
+    activeGameFeaturings,
+    fetchGameFeaturings,
+  });
+
+  const isPlanActive = isMarketingPlanActive(
+    marketingPlan,
+    activeGameFeaturings
+  );
+
+  const requirementsErrors = isPlanActive
+    ? getRequirementsErrors(game, marketingPlan)
+    : [];
+
+  return (
+    <div
+      style={{
+        ...styles.container,
+        backgroundColor: gdevelopTheme.credits.backgroundColor,
+        color: gdevelopTheme.credits.color,
```

Comment:
it feels like you're using the colors of something else for this, should this have its own style props?